.gitignore

```diff
@@ -0,0 +1,2 @@
+# Environment
+venv/
```

Comment:
@ConradKash could you use the other link to python_.gitignore file @cmplx-xyttmt shared
cause ___pycache___ folderz are still being pushed

Reply:
Yeah you are right. I had forgotten thank you.